flang/unittests/RuntimeGTest/Transformational.cpp

```diff
@@ -200,4 +200,20 @@ TEST(Transformational, Unpack) {
     EXPECT_EQ(*result.ZeroBasedIndexedElement<std::int32_t>(j), expect[j]);
   }
   result.Destroy();
+
+  // Test for scalar value of the "field" argument
+  auto scalarField{MakeArray<TypeCategory::Integer, 4>(
```

Comment:
Same question as flang/runtime/transformational.cpp  (@psteinfeld and @schweitzpgi)

Reply:
> 
> 
> Same question as flang/runtime/transformational.cpp (@psteinfeld and @schweitzpgi)

I don't see any similar change on the "main" branch of the llvm-project.

Reply:
OK. So, are you planning on just keeping these changes in fir-dev for now? @schweitzpgi can confirm this...but I assume they would then go into "main"of llvm-project when fir-dev is upstreamed.